plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -329,10 +330,16 @@ export class ElasticSearchSearchEngine implements SearchEngine {
         nextPageCursor,
         previousPageCursor,
       };
-    } catch (e) {
+    } catch (error) {
+      if (error.meta.body.error.type === 'index_not_found_exception') {
```

Comment:
Would it be worth being defensive here? if the error thrown was for any other reason than the query, or if its shape changed, you'd risk getting like `TypeError: Cannot read properties of undefined (reading 'meta')` here. Could do `error.meta?.body?.error?.type` or even wrap the original thing in a nested try/catch in here :)

Reply:
I'll update to check all keys